android-sdk/build.gradle

```diff
@@ -37,9 +38,16 @@ android {
 dependencies {
   compile fileTree(dir: 'libs', include: ['*.jar'])
 
+  compile 'com.madgag.spongycastle:core:1.54.0.0'
```

Comment:
It is used for encryption of Bridge data for uploads. I moved it from researchstack-sdk to android-sdk, since we'll want it in the general Android SDK